### ISSUE TITLE
当上下文长度过长，导致响应时间超过五分钟时，会自动断开，每次都是五分钟准时断开，希望可以增加这个时间

### ISSUE THREAD

**Issue by GitZiyin** [lioensky/VCPToolBox#292]
提示是
[系统错误] VCP流读取错误: terminated

下面是lm studio的错误代码

026-04-19 01:02:08  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Running chat completion on conversation with 10 messages.
2026-04-19 01:02:08  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Streaming response...
2026-04-19 01:02:08 [DEBUG]
 LlamaV4::predict slot selection: session_id=<empty> server-selected (LCP/LRU)
2026-04-19 01:02:09 [DEBUG]
 slot get_availabl: id  2 | task -1 | selected slot by LRU, t_last = -1
srv  get_availabl: updating prompt cache
srv          load:  - looking for better prompt, base f_keep = -1.000, sim = 0.000
srv        update:  - cache state: 0 prompts, 0.000 MiB (limits: 8192.000 MiB, 262144 tokens, 8589934592 est)
srv  get_availabl: prompt cache update took 0.02 ms
slot launch_slot_: id  2 | task -1 | sampler chain: logits -> penalties -> ?dry -> ?top-n-sigma -> top-k -> ?typical -> top-p -> ?min-p -> ?xtc -> ?temp-ext -> dist 
slot launch_slot_: id  2 | task 2530 | processing task, is_child = 0
slot slot_save_an: id  3 | task -1 | saving idle slot to prompt cache
2026-04-19 01:02:09 [DEBUG]
 srv   prompt_save:  - saving prompt with length 37625, total state size = 453.927 MiB
2026-04-19 01:02:09 [DEBUG]
 slot prompt_clear: id  3 | task -1 | clearing prompt with 37625 tokens
2026-04-19 01:02:09 [DEBUG]
 srv        update:  - cache state: 1 prompts, 830.806 MiB (limits: 8192.000 MiB, 262144 tokens, 370993 est)
srv        update:    - prompt 00000297A39715C0:   37625 tokens, checkpoints:  6,   830.806 MiB
slot update_slots: id  2 | task 2530 | new prompt, n_ctx_slot = 262144, n_keep = 0, task.n_tokens = 35123
slot update_slots: id  2 | task 2530 | cache reuse is not supported - ignoring n_cache_reuse = 256
slot update_slots: id  2 | task 2530 | n_tokens = 0, memory_seq_rm [0, end)
2026-04-19 01:02:09 [DEBUG]
 slot update_slots: id  2 | task 2530 | prompt processing progress, n_tokens = 2048, batch.n_tokens = 2048, progress = 0.058309
2026-04-19 01:02:09  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Prompt processing progress: 0.0%
2026-04-19 01:02:31 [DEBUG]
 slot update_slots: id  2 | task 2530 | n_tokens = 2048, memory_seq_rm [2048, end)
2026-04-19 01:02:31 [DEBUG]
 slot update_slots: id  2 | task 2530 | prompt processing progress, n_tokens = 4096, batch.n_tokens = 2048, progress = 0.116619
2026-04-19 01:02:31  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Prompt processing progress: 5.8%
2026-04-19 01:02:50 [DEBUG]
 slot update_slots: id  2 | task 2530 | n_tokens = 4096, memory_seq_rm [4096, end)
2026-04-19 01:02:50  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Prompt processing progress: 11.7%
2026-04-19 01:02:50 [DEBUG]
 slot update_slots: id  2 | task 2530 | prompt processing progress, n_tokens = 6144, batch.n_tokens = 2048, progress = 0.174928
2026-04-19 01:03:11 [DEBUG]
 slot update_slots: id  2 | task 2530 | n_tokens = 6144, memory_seq_rm [6144, end)
2026-04-19 01:03:11 [DEBUG]
 slot update_slots: id  2 | task 2530 | prompt processing progress, n_tokens = 8192, batch.n_tokens = 2048, progress = 0.233237
2026-04-19 01:03:11  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Prompt processing progress: 17.5%
2026-04-19 01:03:30 [DEBUG]
 slot update_slots: id  2 | task 2530 | n_tokens = 8192, memory_seq_rm [8192, end)
2026-04-19 01:03:30 [DEBUG]
 slot update_slots: id  2 | task 2530 | 8192 tokens since last checkpoint at 0, creating new checkpoint during processing at position 10240
slot update_slots: id  2 | task 2530 | prompt processing progress, n_tokens = 10240, batch.n_tokens = 2048, progress = 0.291547
2026-04-19 01:03:30  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Prompt processing progress: 23.3%
2026-04-19 01:03:30 [DEBUG]
 slot update_slots: id  2 | task 2530 | created context checkpoint 1 of 32 (pos_min = 8191, pos_max = 8191, n_tokens = 8192, size = 62.813 MiB)
2026-04-19 01:03:51 [DEBUG]
 slot update_slots: id  2 | task 2530 | n_tokens = 10240, memory_seq_rm [10240, end)
2026-04-19 01:03:51 [DEBUG]
 slot update_slots: id  2 | task 2530 | prompt processing progress, n_tokens = 12288, batch.n_tokens = 2048, progress = 0.349856
2026-04-19 01:03:51  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Prompt processing progress: 29.2%
2026-04-19 01:04:12 [DEBUG]
 slot update_slots: id  2 | task 2530 | n_tokens = 12288, memory_seq_rm [12288, end)
2026-04-19 01:04:12 [DEBUG]
 slot update_slots: id  2 | task 2530 | prompt processing progress, n_tokens = 14336, batch.n_tokens = 2048, progress = 0.408166
2026-04-19 01:04:12  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Prompt processing progress: 35.0%
2026-04-19 01:04:34 [DEBUG]
 slot update_slots: id  2 | task 2530 | n_tokens = 14336, memory_seq_rm [14336, end)
2026-04-19 01:04:34 [DEBUG]
 slot update_slots: id  2 | task 2530 | prompt processing progress, n_tokens = 16384, batch.n_tokens = 2048, progress = 0.466475
2026-04-19 01:04:34  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Prompt processing progress: 40.8%
2026-04-19 01:05:06 [DEBUG]
 slot update_slots: id  2 | task 2530 | n_tokens = 16384, memory_seq_rm [16384, end)
slot update_slots: id  2 | task 2530 | 8192 tokens since last checkpoint at 8192, creating new checkpoint during processing at position 18432
slot update_slots: id  2 | task 2530 | prompt processing progress, n_tokens = 18432, batch.n_tokens = 2048, progress = 0.524784
2026-04-19 01:05:06  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Prompt processing progress: 46.6%
2026-04-19 01:05:07 [DEBUG]
 slot update_slots: id  2 | task 2530 | created context checkpoint 2 of 32 (pos_min = 16383, pos_max = 16383, n_tokens = 16384, size = 62.813 MiB)
2026-04-19 01:05:39 [DEBUG]
 slot update_slots: id  2 | task 2530 | n_tokens = 18432, memory_seq_rm [18432, end)
2026-04-19 01:05:39 [DEBUG]
 slot update_slots: id  2 | task 2530 | prompt processing progress, n_tokens = 20480, batch.n_tokens = 2048, progress = 0.583094
2026-04-19 01:05:39  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Prompt processing progress: 52.5%
2026-04-19 01:06:15 [DEBUG]
 slot update_slots: id  2 | task 2530 | n_tokens = 20480, memory_seq_rm [20480, end)
2026-04-19 01:06:15 [DEBUG]
 slot update_slots: id  2 | task 2530 | prompt processing progress, n_tokens = 22528, batch.n_tokens = 2048, progress = 0.641403
2026-04-19 01:06:15  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Prompt processing progress: 58.3%
2026-04-19 01:06:52 [DEBUG]
 slot update_slots: id  2 | task 2530 | n_tokens = 22528, memory_seq_rm [22528, end)
2026-04-19 01:06:52 [DEBUG]
 slot update_slots: id  2 | task 2530 | prompt processing progress, n_tokens = 24576, batch.n_tokens = 2048, progress = 0.699712
2026-04-19 01:06:52  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Prompt processing progress: 64.1%
2026-04-19 01:07:12  [INFO]
 [LM STUDIO SERVER] Client disconnected. Stopping generation... (If the model is busy processing the prompt, it will finish first.)
2026-04-19 01:07:12 [DEBUG]
 srv          stop: cancel task, id_task = 2530
2026-04-19 01:07:12  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Prompt processing progress: 64.1%
2026-04-19 01:07:12  [INFO]
 [qwen3.6-35b-a3b-uncensored-hauhaucs-aggressive] Finished streaming response
2026-04-19 01:07:23 [DEBUG]
 slot      release: id  2 | task 2530 | stop processing: n_tokens = 24576, truncated = 0
srv  update_slots: all slots are idle

**Comment by lioensky** [comment]
非常神秘确实没有这个限制啊。
